src/log/log.c

```diff
@@ -2387,6 +2388,13 @@ __wt_log_scan(WT_SESSION_IMPL *session, WT_LSN *lsnp, uint32_t flags,
             if (LF_ISSET(WT_LOGSCAN_ONE))
                 break;
         }
+
+        /*
+         * Exit the scanning loop if the next LSN seen is greater than our user set end range LSN.
+         */
+        if (end_lsnp != NULL && __wt_log_cmp(&next_lsn, end_lsnp) == 1)
```

Comment:
Minor: We have a style that instead of checking exactly for `1`, we check ` > 0`.